TTS/auto_tts/example.py

```diff
@@ -0,0 +1,12 @@
+from TTS.utils.manage import ModelManager
```

Comment:
I guess you want to do more here ?

Reply:
yea I added that to the commit by mistake. I was just seeing if I understood the model manager correctly. 
I already added a way to call pretrained models in the AutoTTS repo and I'll add that in here instead of the way I made it download models